lib_eio_luv/eio_luv.mli

```diff
@@ -73,6 +73,19 @@ module File : sig
 
   val mkdir : mode:Luv.File.Mode.t list -> string -> unit or_error
   (** Wraps {!Luv.File.mkdir} *)
+
+end
+
+module Poll : sig
+
+  val poll_readable : Unix.file_descr -> (unit -> unit) -> unit
```

Comment:
```suggestion
  val poll_readable : Unix.file_descr -> unit
```
The function should just suspend the calling fibre until the result is ready. We don't need to use callbacks now we have effects (even if that is convenient for integrating with Lwt's API).